src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -439,7 +439,7 @@ def initial_setup(self):
     def turn_on(self):
         """Turns on the instrument."""
         if not self._is_connected_to_qm:
-            self._qm = self._qmm.open_qm(config=self._config, close_other_machines=False)
+            self._qm = self._qmm.open_qm(config=self._config, close_other_machines=True)
```

Comment:
Do we want to merge this?

Reply:
No, it slipped as I have to put it always for sauron